package.json

```diff
@@ -95,6 +95,7 @@
     "scripts"
   ],
   "overrides": {
-    "mongodb-client-encryption": "^6.1.0"
+    "mongodb-client-encryption": "^6.1.0",
+    "body-parser": "^1.20.3"
```

Comment:
Fwiw, in order to bump a transitive dependency, you don't usually need to put the nested dependency in `overrides`, unless the intermediate dependency (`express` in this case, I guess) has a dependency specification that conflicts with the new version.

It should be enough here to update our `package-lock.json`, i.e. you can remove this line, run `npm install` and the `package-lock.json` should still point to `1.20.3` without issues. That's going to simplify things a bit once our dependencies start depending on body-parser 2.x, for example.

Reply:
Yeah, I think it should be enough to bump express in this case and update package-lock with install, express has an exact version specified for body-parser and express version change should've bumped it in package lock too

Reply:
So this PR is to address the vulnerability in body-parser 1.20.2 (added COMPASS-8288). 

With bump in express, the package lock does update it correctly for compass-web package. But we also have express in compass-oidc and it resolves to body-parser version 1.20.2. 

Running `npm install` with `npm update express` correctly updates the deps.

Reply:
Cool – opened https://github.com/mongodb-js/mongosh/pull/2166 for the mongosh equivalent 🙂

Reply:
Aaaaah, finally got it, I was assuming this whole time that it's only due to express here having the "older" version than the one in oidc-plugin that the error is being triggered. Thanks for clarifying!